RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chrome.java

```diff
@@ -374,6 +372,7 @@ private void getBookmark() {
                                     this.getName(), bookmarkFile.getName()));
                 }
             }
+            postArtifact(bbartifacts);
```

Comment:
Are we posting one artifact or multiple?  If multiple, then this should be a call to postArtifacts (plural).

As it stands, this doesn't build.